airflow-core/src/airflow/migrations/versions/0070_3_1_0_rename_and_change_type_of_deadline_column.py

```diff
@@ -17,7 +17,7 @@
 # under the License.
 
 """
-Change the Deadline column in the Deadline table from DateTime to UTC DateTime.
```

Comment:
Wait, can we modify migrations?  I could certainly be wrong, but I thought these were pretty immutable once they get merged and changing them would break stuff?

Reply:
I think, that's true for any released version. But, since, this is for 3.1.0, we can do this